projects/vdk-core/src/vdk/internal/core/error_classifiers.py

```diff
@@ -0,0 +1,313 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+"""
+errors -- Exception handling implementations
+
+errors is a module that handles all errors.
+It defines classes and methods for handling exceptions, and ensuring that there are two types of exceptions:
+ - infrastructure-related: should be handled by Platform team.
+ - user code-related: should be handled by the owner of the DataJob.
+"""
+from __future__ import annotations
+
+import enum
+import logging
+import re
+from collections import defaultdict
+from enum import Enum
+from logging import Logger
```

Comment:
nit: remove unused and unnecesary imports

Reply:
@antoniivanov  idk why they were firstly imported like that but those imports are probably used in tests because without them the tests fail ijust wanted to put it as it was but I can move them